src/platforms.md

```diff
@@ -1,41 +1,41 @@
 ---
 title: Platforms
-description: "You can use the Dart language to write mobile apps, web apps, standalone apps, servers, and more."
+description: "You can use the Dart SDK to write mobile apps, web apps, standalone apps, servers, and more."
```

Comment:
You use the Flutter SDK, not the Dart SDK, to write mobile apps.

Reply:
Yeah, good point.

I don't like 'Dart language' -- that's to narrow (it's all the surrounding stuff that enables the mobile, web, server, ..., part.

Can we just say 'You can use Dart to write...' ?

Reply:
Still a noun. :(

Reply:
Changing to just Dart as per verbal discussion